src/main/webapp/WEB-INF/datastore-indexes.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<datastore-indexes autoGenerate="false">
```

Comment:
Should we keep it as `autoGenerate="true"` so in the future if new indexes are needed in the future they can be automatically generated as well?